packages/garbo/src/resources/aprilingband.ts

```diff
@@ -17,22 +17,17 @@ const instruments: {
   },
   {
     instrument: "Apriling band saxophone",
-    value: () =>
-      canAdventure($location`Cobb's Knob Treasury`)
-        ? EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")
-        : 0,
+    value: () => getBestLuckyAdventure().value(),
   },
   {
     instrument: "Apriling band piccolo",
     value: () =>
       Math.max(
         0,
-        ...getExperienceFamiliars().map(
-          ({ expectedValue }) => expectedValue / 12,
+        ...getExperienceFamiliars().map(({ familiar, expectedValue }) =>
+          familiar.experience < 281 ? expectedValue / 400 : 0,
```

Comment:
where is 281 coming from?

Reply:
Ahh I think I see--I would bring this back up to 361. It'll give overestimates more often than underestimates, but I still think it's accurate.

It's also worth noting that `expectedValue` is the value _per turn_. The reason I was dividing by 12 is because right now, experience familiars are written with the expectation that they'll get 12 exp per turn.